docs/releases.rst

```diff
@@ -20,10 +21,6 @@ new ``--flavor`` argument. Using this argument the user can choose between a
 ``custom`` flavor where user must provide a custom template using a
 ``--template-path`` argument.
 
-
-tmt-1.37.0
```

Comment:
I *think* this section should exist, just with different name, `1.36.1`.

@psss can you check this? It's basically the bugfix release note we have in 1.36.1 release commits, but not in `main`.

Reply:
Yes, I think you are correct. I did the modifications.